guides/common/assembly_configuring-satellite-custom-server-certificate.adoc

```diff
@@ -25,8 +25,13 @@ To configure your {ProjectServer} with a custom certificate, complete the follow
 . If you have external {SmartProxyServers} registered to {ProjectServer}, configure them with custom SSL certificates.
 For more information, see {InstallingSmartProxyDocURL}configuring-capsule-custom-server-certificate_{smart-proxy-context}[Configuring {SmartProxyServer} with a Custom SSL Certificate] in _{InstallingSmartProxyDocTitle}_.
 
+
```

Comment:
Is this missing "ProductName" similar to https://github.com/theforeman/foreman-documentation/pull/3012/files#diff-b3c1e911c867b76feaa0e96e4320c9057a49ec4d0a776afe750faf65dd3305e2R16?